app/app.cpp

```diff
@@ -1,11 +1,48 @@
 #include "app.h"
 #include <algorithm>
-
+#include<format>
+#include<iostream>
 namespace vsite::oop::v2
 {
+
 /*
 * class member function implementations
 */
+	color::color(double red, double green, double blue) {
+		this->r = red;
+		this->g = green;
+		this->b = blue;
+	}
+
+	double color::get_red() const {
+		return r;
+	}
+	double color::get_green() const {
+		return g;
+	}
+	double color::get_blue() const {
+		return b;
+	}
+	void color::set_red(double red) {
+		this->r = std::clamp(red, 0., 1.);
+	}
+	void color::set_green(double green) {
+		this->g = std::clamp(green, 0., 1.);
+	}
+	void color::set_blue(double blue) {
+		this->b = std::clamp(blue, 0., 1.);
+	}
+	std::string to_hex(int num) {
```

Comment:
Zašto je ovo član klase, kakve to veze ima sa bojom (prima neki cijeli broj)? Stavite je direktno u `main.cpp`.